opentelemetry-sdk/src/opentelemetry/sdk/_metrics/view.py

```diff
@@ -0,0 +1,172 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=protected-access
+
+from dataclasses import dataclass
+from logging import getLogger
+from re import match
+from typing import Dict
+
+from opentelemetry._metrics.instrument import Instrument
+from opentelemetry.sdk._metrics.aggregation import Aggregation
+
+_logger = getLogger(__name__)
+
+
+@dataclass
+class View:
+    def __init__(
+        self,
+        instrument_type: Instrument = None,
```

Comment:
```suggestion
        instrument_type: Optional[Type[Instrument]] = None,
```

Reply:
It looks like all of these fields are `Optional`

Reply:
Fixed all :+1: